tests/ui/impl-trait/dyn-trait-return-should-be-impl-trait.stderr

```diff
@@ -148,8 +148,8 @@ LL |         return Struct;
    |
    = note: expected struct `Box<(dyn Trait + 'static)>`
               found struct `Struct`
-   = note: for more on the distinction between the stack and the heap, read https://doc.rust-lang.org/book/ch15-01-box.html, https://doc.rust-lang.org/rust-by-example/std/box.html, and https://doc.rust-lang.org/std/boxed/index.html
-help: store this in the heap by calling `Box::new`
+   = note: for more on the distinction between the stack and the heap, see: https://doc.rust-lang.org/book/ch15-01-box.html, https://doc.rust-lang.org/rust-by-example/std/box.html, and https://doc.rust-lang.org/std/boxed/index.html
+help: try storing this on the heap by calling `Box::new`
```

Comment:
I don't really like these drive-by tweaks of diagnostic messages. It makes the PR larger and doesn't really make things better IMO.

Reply:
that is fair, fixed in newest commit.